dom.go

```diff
@@ -136,7 +136,14 @@ func (e Node) buildHTML(sb *strings.Builder) *strings.Builder {
 		sb.WriteString(" ")
 		attr.buildHTML(sb)
 	}
+
+	if e.isVoidElement() {
+		sb.WriteString("/>")
+		return sb
+	}
+
 	sb.WriteString(">")
+
```

Comment:
if `buildHTML` is the only place we need this info, i'd rather inline it as a `switch`

```suggestion
	switch e.Name {
		case "area", "base", "br", "col", "command", "embed", "hr", "img", "input", "keygen", "link", "meta", "param", "source", "track", "wbr":
			sb.WriteString("/>")
			return
		default:
			sb.WriteString(">")
	}
```

Reply:
okay, updated!